src/services/__tests__/dealer.test.ts

```diff
@@ -13,32 +13,32 @@ describe("Dealer", () => {
   describe("#fetchDealerSuggestions", () => {
     it("encodes the query", async () => {
       const query = "k+k"
-      await fetchDealerSuggestions(query)
+      await fetchDealerSuggestions({ query })
```

Comment:
yeah, with pretty much all of these, I struggle seeing the value in the change. can we get some other opinions?

Reply:
Let's keep the discussion about it here: https://github.com/carforyou/carforyou-api-client-pkg/pull/179#discussion_r505456515